bin/configs/java-jersey2-8.yaml

```diff
@@ -8,3 +8,4 @@ additionalProperties:
   serverPort: "8082"
   dateLibrary: java8
   useOneOfDiscriminatorLookup: true
+  disallowAdditionalPropertiesIfNotPresent: false
```

Comment:
@wing328 , I think at some point we should set `disallowAdditionalPropertiesIfNotPresent` to false by default, maybe starting with jersey2.
Historically, the behavior has been to disallow additional properties by default, but that's not what the spec states.

Reply:
I think we should let the user choose at this point instead of forcing the behavior per generator/library.

We'll introduce this option to the community as part of the v5.0.0 official release and consider changing the default value later if more users are familiar with the change.

Reply:
ok, sure. This is how it is currently implemented (i.e. non-compliant). I have set disallowAdditionalPropertiesIfNotPresent to false for the example only.